Update deps

pablocostass opened the pull request:
Update deps again in order to get the fix for the timeouts in `excoveralls`.

PragTob commented:
Also, thanks for the PR @pablocostass !

the appveyor/windows build failed... tried restarting it. We'll see.

pablocostass commented:
It failed again with the same error while compiling `hackney_ssl`.
```erlang
===> Compiling src/hackney_ssl.erl failed
src/hackney_ssl.erl:58: {undef,[{ssl,cipher_suites,[default,'tlsv1.2'],[]},
        {erl_eval,do_apply,6,[{file,"erl_eval.erl"},{line,670}]},
        {erl_eval,eval_lc1,6,[{file,"erl_eval.erl"},{line,696}]},
        {erl_eval,eval_generate,7,[{file,"erl_eval.erl"},{line,725}]},
        {erl_eval,eval_lc,6,[{file,"erl_eval.erl"},{line,682}]},
        {erl_eval,expr,5,[{file,"erl_eval.erl"},{line,438}]},
        {erl_eval,exprs,5,[{file,"erl_eval.erl"},{line,122}]},
        {ct_expand,xform_fun,6,
                   [{file,"c:/projects/benchee-8e4gw/deps/parse_trans/src/ct_expand.erl"},
                    {line,95}]}]}
```

It says that `ssl:chiper_suites/2` is undefined, but it was added back in OTP 20.3 and Chocolatey is downloading Erlang/OTP 22.3 :thinking:

PragTob commented:
It's so strange... I started a new build on the default branch to see if it has anything to do with the PR or is just generally broken: https://ci.appveyor.com/project/PragTob/benchee-8e4gw

PragTob commented:
Welp I might have to migrate it all to github actions so that we can have mac builds anyhow. Fun, fun times. #needtimeforopensource

pablocostass commented:
It's weird indeed, it really does seem broken. About the Github Actions, I can give migrating to them a try if you don't mind :)

PragTob commented:
@pablocostass gladly, I'm always happy for more help :tada: I'm just not sure how that'd work full circle since I guess we/I have to activate it on the repo. But I guess you can activate it on your fork first to develop it and then we can activate it here?

So yeah, it'd be much appreciated. We can start with linux & Windows to replace what we currently have. Down the line, I think github has macos support which I'd love to add. I'd wait with that, I have a hunch that one test may be failing there but hopefully not :)

pablocostass commented:
@PragTob okay, [I've added them to my fork](https://github.com/pablocostass/benchee/actions/runs/146503030).

Could you review them and see if there's anything to improve/change? Also, I went ahead and added MacOS too to see if it would work or not, and you were totally right, there's a test that fails on it! It's because the following test expects an output containing `"1.00x memory"` but in MacOS it seems to be `"1.04x memory"`:
```elixir
test memory measurement the micro keyword list code from Michal does not break memory measurements #213 (BencheeTest)
     test/benchee_test.exs:767
     Assertion with =~ failed
     code:  assert output =~ "1.00x memory"
     left: [...] # the whole output
     right: "1.00x memory"
```
I'll wait for your input before doing anything else. I can delete the MacOS job if you want, but it's a quick fix otherwise: the assert should be 
```elixir
assert output =~ "1.00x memory" || output =~ "1.04x memory"
```

PragTob commented:
@pablocostass argh sorry, time again :( 

That's great sounding/looking!

just for the suggested fix, I'd prefer something like:

```ruby
if System.mac_os? do # we have code for this in the System module, I haven't looked it up right now but the equivalent of it
  output =~ ...
else
  # ...
end
```

Seems a tiny bit better thant his to me. Please PR :)

PragTob commented:
@pablocostass alright, I guess we can finally get back here, so after rebasing you should be good here. If you don't have the time I can do it as well one of these days :)

pablocostass commented:
@PragTob done :)
